include/openssl/types.h

```diff
@@ -6,10 +6,23 @@
  * in the file LICENSE in the source distribution or at
  * https://www.openssl.org/source/license.html
  */
+ 
+/*
+ * Unfortunate workaround to avoid symbol conflict with wincrypt.h
+ * See https://github.com/openssl/openssl/issues/9981
+ */
+# ifdef _WIN32
+#  define WINCRYPT_OPENSSL_SYMBOL_COMPAT
```

Comment:
Why define this?

Reply:
The intent is that I can update wincrypt.h in future to add a prefix to the conflicting symbols on the wincrypt side, if this symbol is present. This means in the long term (probably several years away) we can remove these undefs.